deliverables/ruby_monk_problems/exec_time.rb

```diff
@@ -1,3 +1,11 @@
+=begin
+You are given some code in the form of lambdas. 
+Measure and return the time taken to execute that code. 
+You may use Time.now to get the current time.
+=end
+
 def exec_time(proc)
-  # your code here
+  	begin_time = Time.now
+  	proc.call
+  	Time.now - begin_time
 end
```

Comment:
Maybe here we can create a proc locally just to try to run this locally. Can you think of a way to do that?